tests/test_series.py

```diff
@@ -1236,3 +1236,32 @@ def cond2(x: pd.Series[int]) -> pd.Series[bool]:
 
     cond3 = pd.Series([False, True, True])
     check(assert_type(s.where(cond3, other=0), "pd.Series[int]"), pd.Series, int)
+    
+
+def test_bitwise_opeator() -> None:
```

Comment:
use `test_bitwise_operator()`  it is misspelled

Reply:
done sir